CMSgov/onemac
# OY2-13442: Ingest RAI data obtained from SEATool in OneMAC

Story: https://qmacbis.atlassian.net/browse/OY2-13442
Endpoint: https://d37vvg9moxp7a1.cloudfront.net/

### Details
RAI data coming in from SEATool via BigMAC gets added to the OneMAC data

### Changes
- added the RAI topic to the list of topics that OneMAC will receive events from
- because submit.js is part of this PR, added unit testing to many of its function calls

### Implementation Notes
Because this RAI topic data is not yet used by the Front End, the storage structure may need tweaking to support the UI needs.  At this time, it is a data dump of the fields with a logical way to access them.

Events from the RAI topic are dependent on the BigMAC environment, which is out of our control.  We have received events, so can use those for testing and verifying this PR.

### Test Plan
1. Testing this story requires access to the AWS Console.
2. Login to Macstack Dev Web Access in AWS
3. Go to CloudWatch->Log Groups and select "/aws/lambda/stream-functions-add-rai-sinkMskToDynamo" group
4. Click the top stream
5. scroll through the logs and select a line with "Received event:"  Expand the event and verify that "topic": "lmdc.seatool.submission.cdc.sea-dbo-RAI" is part of the event.
6. Make note of the payload details for that event, should include:
  a. ID_Number. (required)
  b. [RAI_Requested_Date] (required)
  c. [RAI_Received_Date] (optional - null if absent)
  d. [RAI_Withdrawn_Date] (optional - null if absent)
7. Notice the Offset value as well (this is essentially the event id according to my reasearch)
8. Open DynamoDB in console, travel to Items, and select "onemac-add-rai-one" table.
9. query the table with pk being the ID_Number from the event (it does not matter if this RAI is for a package we care about or not)
10. Open the item with sk of SEATool#RAI#<offset>
11. Verify that the fields contain the same values as the event. (I do not know why AWS capitalizes Null)
